controllers/user/views/update-email.njk

```diff
@@ -0,0 +1,44 @@
+{% from "components/form-fields/macros.njk" import formErrors, formField with context %}
+{% from "components/breadcrumb-trail/macro.njk" import breadcrumbTrail %}
+
+{% extends "layouts/main.njk" %}
+
+{% set title = 'Update email address' %}
+{% set bodyClass = 'has-static-header' %}
+
+{% block content %}
+    <main role="main" id="content">
+        <div class="content-box u-inner-wide-only">
+            {{ breadcrumbTrail(breadcrumbs) }}
+
+            <h1>{{ title }}</h1>
+            <p>You can modify your registered email address here. Please bear in mind you will need to confirm your account again once you have done this.</p>
+
+            {% if alertMessage %}
+                <div class="message" role="alert">
+                    {{ alertMessage }}
+                </div>
+            {% endif %}
+
+            <form action="" method="post" autocomplete="off">
+                {{ formErrors(errors = errors, title = "There was a problem with your submission") }}
+
+                {% if csrfToken %}
+                    <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+                {% endif %}
+
+                {{ formField({
```

Comment:
Is this enough to ask for to change the email address? What's an appropriate pattern here, should we be asking to for password confirmation to do this? We can come back to this in a later PR.